src/Components/Navigation.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { FaUser } from 'react-icons/fa';
+
+const Navigation = () => (
+  <>
+    <div className="nav-bar">
+      <div className="logo">
+        <p className="magicians">Bookstore CMS</p>
+      </div>
+      <nav className="nav">
+        <Link className="links" style={{ textDecoration: 'none' }} to="/">Books</Link>
+        <Link className="links" style={{ textDecoration: 'none' }} to="/categories">Categories</Link>
+      </nav>
+      <FaUser className="icon" />
+    </div>
+  </>
+);
+
```

Comment:
- [ ] As mentioned by the previous reviewer to stick as much as possible to template design, I think you should improve your header design to match the expected result e.g.: color text, width
 of the header block, spacing around header elements
 ### Template design
![Screenshot from 2022-12-04 01-20-56](https://user-images.githubusercontent.com/83042742/205466943-b957605c-e189-40b9-bbc9-be1148585b3e.png)

 ### Your design
![Screenshot from 2022-12-04 01-44-06](https://user-images.githubusercontent.com/83042742/205467020-b21bdaf2-fa57-4839-9d86-0c0a0893c2af.png)